ops/charm.py

```diff
@@ -525,7 +740,17 @@ def __init__(self, role: RelationRole, relation_name: str, raw: dict):
 
 
 class StorageMeta:
-    """Object containing metadata about a storage definition."""
+    """Object containing metadata about a storage definition.
+
+    Attributes:
```

Comment:
the list below attributes needs to be indented afaik

Reply:
fixed